### PR TITLE
misc: Add design decision about bitnami-shell

### PR DESCRIPTION
**Pull request by jotadrilo**
Please find a proper description in the change itself.

### CODE REVIEW

File: DESIGN_DECISIONS.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ Use the following format for contributing to this document:
   <Reasoning and explanation, putting any related issues and PRs (if any)>
 ```
 
+## 2021-02-25: Replace `bitnami/minideb` by `bitnami/bitnami-shell`
+
+**TL;DR:** [`bitnami/minideb`](https://github.com/bitnami/minideb) is not used in initContainers anymore. [`bitnami/bitnami-shell`](https://github.com/bitnami/bitnami-docker-bitnami-shell) will be used from now on, instead.
```
Comment (juan131) [inline DESIGN_DECISIONS.md:17]:
```suggestion
**TL;DR:** [`bitnami/minideb`](https://github.com/bitnami/minideb) is not used on auxiliar containers (such as init containers or sidecar containers) anymore. [`bitnami/bitnami-shell`](https://github.com/bitnami/bitnami-docker-bitnami-shell) will be used from now on, instead.
```

File: DESIGN_DECISIONS.md [hunk 2]
```diff
@@ -12,6 +12,14 @@ Use the following format for contributing to this document:
...
+
+The [`bitnami/minideb`](https://github.com/bitnami/minideb) image started being a good fit for initContainers (see [why use minideb](https://github.com/bitnami/minideb#why-use-minideb) section).
```
Comment (juan131) [inline DESIGN_DECISIONS.md:19]:
```suggestion
The [`bitnami/minideb`](https://github.com/bitnami/minideb) image started being a good fit for auxiliar containers (see [why use minideb](https://github.com/bitnami/minideb#why-use-minideb) section).
```

File: DESIGN_DECISIONS.md [hunk 3]
```diff
@@ -12,6 +12,14 @@ Use the following format for contributing to this document:
...
+However, several initContainers required non trivial packages to be present (e.g. `sysctl`). We can argue that many of these requirements would be a good-to-have. Nevertheless, we aim to keep [`bitnami/minideb`](https://github.com/bitnami/minideb) very light-weight and, more important, with a very small vulnerability surface. The more packages you add, the more vulnerabilities the image can have.
```
Comment (juan131) [inline DESIGN_DECISIONS.md:20]:
```suggestion
However, several these containers required non trivial packages to be present (e.g. `sysctl`). We can argue that many of these requirements would be a good-to-have. Nevertheless, we aim to keep [`bitnami/minideb`](https://github.com/bitnami/minideb) very light-weight and, more important, with a very small vulnerability surface.
```

**Review by juan131 — APPROVED** [review]
LGTM

**Review by carrodher — APPROVED** [review]
Thanks for your changes. LGTM

**Pull request merged**